driver/parser/parser_test.go

```diff
@@ -194,8 +192,52 @@ applications:
         period: 120
 `
 
-func TestParseWithGenesisWorks(t *testing.T) {
-	_, err := ParseBytes([]byte(withGenesis))
+func TestParseWithGenesisImportWorks(t *testing.T) {
+	_, err := ParseBytes([]byte(withGenesisImport))
+	if err != nil {
+		t.Fatalf("parsing of input failed: %v", err)
+	}
+}
+
+// withGenesisImport extends the small scenario with genesis option.
+var withGenesisExport = `
+name: Small Test With Genesis
+num_validators: 5
+nodes:
+  - name: A
+    instances: 10
+    features:
+      - validator
+      - archive
+    start: 5
+    end: 7.5
+    genesis:
+      export: /path/to/genesis
```

Comment:
Theoretically, same as `genesis.import` case.

Reply:
Pretty much. I've just wanted to test and make sure that syntax of yaml is correct.

Reply:
Same as above, this is simply testing the parser